### PR TITLE
Add prerelease-prefix parameter

### PR DESCRIPTION
**Pull request by thorgull**

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -42,6 +43,7 @@ func Execute(version FullVersion) error {
 		DropTagNamePrefix: *dropPrefixFlag,
 		PrereleaseSuffix:  *prereleaseSuffixFlag,
 		Format:            *formatFlag,
+		PrereleasePrefix:  *prereleasePrefixFlag,
```
Comment (choffmeister) [inline cmd/root.go:45]: For sake of consistent ordering maybe move it up one line (suffix, prefix, format) to match the ordering of the flags above

**Review by choffmeister — CHANGES_REQUESTED** [review]
Thanks for the PR. LGTM except for one minor thing

File: cmd/root.go (hunk 1 above)
Comment (thorgull) [reply cmd/root.go:45]: I changed order both in caller (Execute method,) and in the type definition.

**Review by choffmeister — APPROVED** [review]

**Comment by choffmeister** [comment]
@thorgull Will be released as `v0.3.4` in a few minutes, see https://github.com/choffmeister/git-describe-semver/actions/runs/1744228449

**Pull request merged**